perun-openapi/openapi.yml

```diff
@@ -149,6 +149,16 @@ components:
       discriminator:
         propertyName: beanName
 
+    AssignedMember:
+      allOf:
+        - $ref: '#/components/schemas/Member'
```

Comment:
I think "_allOf_" should be used in case of inheritance, see how _AssignedGroup_ or _AssignedResource_ objects are defined in this file.

Reply:
fixed